test/reporters/base.spec.js

```diff
@@ -507,21 +507,34 @@ describe('Base reporter', function () {
     );
   });
 
-  describe('when reporter output immune to user test changes', function () {
-    var baseConsoleLog;
+  it('should list all the errors within an AggregateError', function () {
+    var err1 = new Error('1');
+    var err2 = new Error('2');
+    var aggErr = new AggregateError([err1, err2], '2 errors');
 
-    beforeEach(function () {
-      sinon.restore();
-      sinon.stub(console, 'log');
-      baseConsoleLog = sinon.stub(Base, 'consoleLog');
-    });
+    var test = makeTest(aggErr);
+    list([test]);
 
-    it('should let you stub out console.log without effecting reporters output', function () {
-      Base.list([]);
-      baseConsoleLog.restore();
+    var errOut = stdout.join('\n').trim();
 
-      expect(baseConsoleLog, 'was called');
-      expect(console.log, 'was not called');
-    });
+    expect(errOut, 'to contain', ' Error: 1', 'Error: 2');
```

Comment:
[Testing] Aggregate / multiple test error reports can be tricky. I think we'd want to test more than just whether these two lines are included. Could you test more directly what the actual output is? I.e. that it includes the proper newlines, spacing, any details/messages/stacks (or not) as expected?